python/robot-framework/tests/login_tests/login.robot

```diff
@@ -0,0 +1,26 @@
+*** Settings ***
+Documentation     A test suite with a single test for valid login.
+...
+...               This test has a workflow that is created using keywords in
+...               the imported resource file.
+Resource          resource.robot
+
+*** Test Cases ***
+Valid Login
+    Open Browser To Login Page
+    ${username} =     Setup User
+    Input Username    ${username}
+    Input Password    secret_sauce
+    Submit Credentials
+    Welcome Page Should Be Open
+    Visual Snapshot    Valid Login    capture_dom=True
```

Comment:
Maybe a bit late in the process, but shouldn't we use `Sauce Visual Snapshot` (or other declension like `Sauce Visual Check` / etc...) ?

We used to in other frameworks prefix with `Sauce` when the function is in a global scope (like wdio).

Reply:
IMO that just makes it more verbose? In theory if there were global keyword conflicts in Robot Framework they could use an aliased / namespaced version like `SauceLabsVisual.Visual Check` to specify ours

Reply:
No need then